pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc-search-keyword.adoc

```diff
@@ -0,0 +1,9 @@
+[id='search-keyword_{context}']
+
+The Search tab allows you to enter keywords or free text to search for modules.
+
+.Procedure
+
+. In the search bar at top right, enter a free text phrase or keywords.
+. You can add Boolean operators to the keywords to refine the search.
```

Comment:
I don't think boolean operators are supported, unless I misunderstand what you mean here.

I searched "Assembly" on my system with 20 documents and got 1 result.

I searched for "-Assembly" and expected to get the other 19, but I got 0.